code/datums/preferences.dm

```diff
@@ -191,23 +217,27 @@ datum/preferences
 		if (AH)
 			dat += "<a href='byond://?src=\ref[user];preferences=1;s_tone=input'><b>Skin Tone:</b></a> [-AH.s_tone + 35]/220<br>"
 
-			dat += "<a href='byond://?src=\ref[user];preferences=1;underwear=input'><b>Underwear:</b></a> [AH.underwear] "
-			dat += "<a href='byond://?src=\ref[user];preferences=1;underwear_color=input'><font face=\"fixedsys\" size=\"3\" color=\"[AH.u_color]\"><b>#</b></font></a><br>"
-
 			dat += "<a href='byond://?src=\ref[user];preferences=1;eyes=input'><b>Eye Color:</b> <font face=\"fixedsys\" size=\"3\" color=\"[AH.e_color]\"><b>#</b></font></a><br>"
+			
+			dat += "<div>Underwear:"
+			dat += generate_select_table("underwear", AH.underwear, underwear_styles)
+			dat += "<a href='byond://?src=\ref[user];preferences=1;underwear_color=input'><font face=\"fixedsys\" size=\"3\" color=\"[AH.u_color]\"><b>#</b></font></a><br></div>"

```

Comment:
Use single quotes for attributes in this section, e.g. `<font face='fixedsys'>`

Reply:
That part was not written by me, it just looks like an addition because of the surrounding stuff for git. I'm pretty sure that all the other links used the same style.